linux/org.raspberrypi.rpi-imager.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--Copyright 2020 Flathub maintainers-->
+<component type="desktop">
+  <id>org.raspberrypi.rpi-imager</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Raspberry Pi Imager</name>
+  <summary>Raspberry Pi imaging utility</summary>
+  <description>
+    <p>
+      Raspberry Pi Imager downloads a .JSON file from our website with a list of all current download options, ensuring you are always installing the most up-to-date version.
```

Comment:
Should this say "the Raspberry Pi website" rather than "our website" ?

Reply:
Fixed

Reply:
Now that you've added word-wrapping, I can see there's also a "our website" in the 2nd paragraph :wink:

Reply:
Fixed